instagram/src/components/App/App.css

```diff
@@ -0,0 +1,15 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+
```

Comment:
This is mostly a very minor issue, but a linter may see an error when there is more than 1 line space between sections.